app/templates/base/home.html

```diff
@@ -74,7 +74,7 @@ <h3 class="heading-small">Data access levels</h3>
                     {% endif %}
                   </td>
                   <td class="align-right no-wrap">
-                    <a class="button button-secondary" href="{{ url_for('buckets.aws', { id: users3bucket.s3bucket.id }) }}">View data in AWS</a>
+                    <a class="button button-secondary" href="{{ url_for('buckets.aws', { id: users3bucket.s3bucket.id }) }}">Open AWS</a>
```

Comment:
This should check something like `if users3bucket.s3bucket.location_url or config.aws.login_url` then show a link, else don't show a link.

Reply:
If we have to show a link, then not having `config.aws.login_url` set is an exception and should be raised as early as possible.